staff/staff_views_helper.py

```diff
@@ -24,8 +40,18 @@ def create_user_with_profile(form, random_password):
     return new_user
 
 
+def validate_username(username):
+    i = 1
+    while DjangoUser.objects.filter(username=username).count() > 0:
+        username = username + str(i)
+        i += 1
+    return username
+
+
 def parse_new_user(new_user, form):
     birthdate = form.cleaned_data.get("birthdate")
+    if birthdate is None:
+        birthdate = datetime.strptime("January 1, 1901", "%B %d, %Y")
```

Comment:
I have the default birthdate because the way that I reference a user is by their salesforce ID, which defaults to the first three letters of the first name, first three of last name, and birthdate.  If a user doesn't enter their birthdate during registration though, I default it to that.  I could probably also just make birthdate a required field on the form.